include/natalie/gc/heap.hpp

```diff
@@ -19,6 +19,8 @@ class Heap {
 public:
     NAT_MAKE_NONCOPYABLE(Heap);
 
+    static long long allocations;
+
```

Comment:
Was this just for debugging? Can it be removed?

Reply:
Oh yeah I was debugging allocations by counting them...  Thought I removed this already 🤔